ush/mesoscale/ush_sref_plot_cnv_py/df_preprocessing.py

```diff
@@ -0,0 +1,306 @@
+###############################################################################
```

Comment:
For python scripts you must use the following in the first line:
`#! /usr/bin/env python3`
NCO standards requires the interpreter be stated in line 1.